docs/sdm/maxent.md

```diff
@@ -14,9 +14,7 @@ Formally, Maxent estimates habitat suitability (i.e. the fundamental niche) usin
 
 Maxent doesn't directly estimate relationships between presence/background data and environmental covariates (so, not just `y ~ x`). Instead, it fits a series of feature transformatons to the covariate data (`z = f(x)`), like computing quadratic transformations or computing the pairwise products of each covariate. Maxent then estimates the conditional probability of finding a species given a set of environmental conditions as:
 
-```
-Pr(y = 1 | f(z)) = (f_1(z) * Pr(y = 1)) / f(z)
-```
+$$Pr(y = 1 | f(z)) = (f_1(z) * Pr(y = 1)) / f(z)$$
```

Comment:
This seemed formatted for math syntax but in a code block, so I thought you might prefer it like this

Reply:
ah, yes, good idea. I'll have to add some additional stuff so `mkdocs` can render it well but I'll add that in another branch.